tests-perf/locust/locust-notifications.py

```diff
@@ -1,24 +1,88 @@
+""" locust-notifications.py
+   isort:skip_file
+"""
+# flake8: noqa
+
 import os
+import sys
+from datetime import datetime
+
+sys.path.append(os.path.abspath(os.path.join("..", "tests_smoke")))
 
 from dotenv import load_dotenv
 from locust import HttpUser, constant_pacing, task
+from tests_smoke.smoke.common import job_line, rows_to_csv  # type: ignore
 
 load_dotenv()
-AUTH_HEADER = os.getenv("TEST_AUTH_HEADER")
 
 
 class NotifyApiUser(HttpUser):
 
     wait_time = constant_pacing(60)
-    host = "https://api.staging.notification.cdssandbox.xyz"
+    host = os.getenv("LOAD_TEST_DOMAIN", "https://api.staging.notification.cdssandbox.xyz")
+
+    def __init__(self, *args, **kwargs):
+        super(NotifyApiUser, self).__init__(*args, **kwargs)
+
+        self.headers = {"Authorization": os.getenv("TEST_AUTH_HEADER")}
+        self.email = "success@simulator.amazonses.com"
+        self.template_id = "5ebee3b7-63c0-4052-a8cb-387b818df627"
+
+    @task(1)
+    def send_email_notifications(self):
+        json = self.__email_json("a59b313d-8de2-4973-ac2f-66de7ec0b239")
+
+        self.client.post("/v2/notifications/email", json=json, headers=self.headers)
+
+    @task(2)
+    def send_email_with_attachment_notifications(self):
+        personalisation = {
+            "application_file": "arbitrary text for this variable",
+            "attached_file": {
+                "file": "Q29udGVudCBvZiBBdHRhY2hlZCBmaWxl",
+                "filename": "attached_file.txt",
+                "sending_method": "attach",
+            }
+        }
+        json = self.__email_json(self.template_id, personalisation)
+
+        self.client.post("/v2/notifications/email", json=json, headers=self.headers)
 
-    @task
-    def send_notifications(self):
-        headers = {"Authorization": AUTH_HEADER}
+    @task(4)
+    def send_email_with_link_notifications(self):
+        personalisation = {
+            "application_file": {
+                "file": "Q29udGVudCBvZiBBdHRhY2hlZCBmaWxl",
+                "filename": "attached_file.txt",
+                "sending_method": "link",
+            }
+        }
+        json = self.__email_json(self.template_id, personalisation)
+
+        self.client.post("/v2/notifications/email", json=json, headers=self.headers)
+
+    @task(8)
```

Comment:
This could have less weight as a task, 1 or 2 I think.

Reply:
Ok will update it